numpy/core/tests/test_cpu_features.py

```diff
@@ -103,6 +172,135 @@ def load_flags_auxv(self):
                     hwcap_value[1].upper().decode().split()
                 )
 
+    def test_runtime_feature_selection(self):
```

Comment:
```suggestion
class TestEnvPrivation:
    def test_runtime_feature_selection(self):
```
Moves the new test methods into a separate class, `AbstractTest` was exist to serve only one purpose which load, parse CPU features at OS level, and only supports linux at current moment, while the new test cases should covers all operating systems.